src/blocks/net.rs

```diff
@@ -36,11 +38,24 @@ impl NetworkDevice {
         let wireless = device_path.join("wireless").exists();
         let tun = device_path.join("tun_flags").exists() || device.starts_with("tun") || device.starts_with("tap");
 
+        fn is_wg_device(wg_uevent_path: &PathBuf) -> bool {
+            if wg_uevent_path.exists() {
+                let uevent_data = read_to_string(&wg_uevent_path).expect("Unable to read file");
```

Comment:
In general we want to avoid `panic`-ing, is it possible to use `?` here instead of `expect()`?

Reply:
Actually, since we don't return a `Result` here, maybe just treat failures to read this file as `false` for the Wireguard check.

Reply:
Read some examples with error handling - this way should be more concise, and without panicking since any error is interpreted as `false`
Does it satisfy our requirements?
```rust
        let wg_uevent_path = device_path.join("uevent");
        let wg = match read_to_string(&wg_uevent_path) {
                Ok(s) => s.contains("wireguard"),
                Err(_e) => false,
        };
```

Reply:
Looks good to me.